src/operator/ResourcesCloudProvider.tsx

```diff
@@ -0,0 +1,86 @@
+import {
+  ComponentColor,
+  DropdownMenuTheme,
+  MultiSelectDropdown,
+} from '@influxdata/clockface'
+import React, {FC, useContext} from 'react'
+import {OperatorContext} from './context/operator'
+import {OperatorRegions} from 'src/types'
+
+const ResourcesCloudProvider: FC = () => {
```

Comment:
I think the name of this should change, since in the context system we use, *Provider refers to a [context provider](https://reactjs.org/docs/context.html#contextprovider). Usually something with the postfix of `Provider` wraps components that consume a react context.

Reply:
Ah yeah, I didn't think about the context providers. I went ahead and changed this to `CloudClusters` instead.